airflow/models/taskinstance.py

```diff
@@ -214,7 +214,8 @@ def clear_task_instances(
             task_id = ti.task_id
             if dag and dag.has_task(task_id):
                 task = dag.get_task(task_id)
-                ti.refresh_from_task(task)
+                # Preserve original assigned queue when clearing a task
+                ti.refresh_from_task(task, queue_override=task.queue)
```

Comment:
A stupid question, what is the difference between extracting `queue` from task then sending it to `refresh_from_task` to use it as queue, and extracting it from the same task in the method `refresh_from_task`? I don't see how this can preserve queue on task clear :thinking:

Reply:
Not a stupid question 😄 , this is error on my side, should be passing `ti.queue` and not `task.queue`